DragonSlayer.cpp

```diff
@@ -1,16 +1,32 @@
 #include "DragonSlayer.h"
 #include "Dragon.h"
+#include "Utility.h"
 
-//DragonSlayer::DragonSlayer
+DragonSlayer::DragonSlayer(const std::string& name_, int hp_, int armor_) :
+    Character(hp_, armor_, 4),
+    name(name_),
+    attackItem(std::unique_ptr<AttackItem>(new AttackItem()))
+    {
+        defensiveItems = makeDefensiveItems(2);
+        helpfulItems = makeHelpfulItems(1);
+    }
```

Comment:
All of this should be aligned with the left edge of the editor
- [ ] revise: formatting